timetracker/vms/test/views/test_employee_detail_view.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+
+
+@pytest.mark.integration
+def test_GET_as_client_admin(client, client_admin_factory, employee_factory):
+
+    admin = client_admin_factory()
+    client.force_login(admin.user)
+
+    employee = employee_factory(client=admin.client)
+
+    url = employee.get_absolute_url()
+    response = client.get(url)
+
+    assert response.status_code == 200
+    assert response.context_data['is_client_admin']
+    assert not response.context_data['is_employee']
+    assert response.context_data['unapproved_count'] == (
```

Comment:
Think about making this a more concrete assertion. If you don't want to, that's fine too. Both ways have their merits.